include/courtroom.h

```diff
@@ -318,6 +318,9 @@ class Courtroom : public QMainWindow {
 
   void on_authentication_state_received(int p_state);
 
+  int get_jud() { return is_jud; }
+  void set_jud(int jud) { is_jud = qMax(-1, qMin(jud, 1)); }
```

Comment:
What does this mean? Why is `is_jud` clamped to -1, 1?

Reply:
because the only valid values are -1, 0, and 1

Reply:
And what does `jud` mean?

Reply:
see https://github.com/AttorneyOnline/docs/blob/judgebuttons-2.9.1/docs/development/network.md#judge-controls

Reply:
made the variable names less confusing, see https://github.com/AttorneyOnline/AO2-Client/pull/537/commits/12ad13629002d04794b0d68ebc28382da4592d5f